doc/releases/changelog-dev.md

```diff
@@ -17,6 +17,8 @@
 * Alias for Identity (`I`) is now accessible from `qml.ops`.
   [(#7200)](https://github.com/PennyLaneAI/pennylane/pull/7200)
 
+* Snapshot specific shots can now be passed to `qml.Snapshot`. 
```

Comment:
```suggestion
* `qml.Snapshot` can now accepts its own `shots` argument for inspecting finite-shot measurements in a circuit.
  [(#7326)](https://github.com/PennyLaneAI/pennylane/pull/7326)
  
  The value for `shots` provided to `qml.Snapshot` overrides the `shots` value specified in the `device` or elsewhere.
```

Is that last sentence correct?

An example would be nice to have here, too! The one in the docstring (maybe reduced by a few lines / operations) looks nice 👀

Reply:
I'm not sure this is a big enough feature to go into much detail.  I don't want to clutter the changelog. I'm happy to say "see the docstring for `qml.Snapshot` for more information?

Reply:
```suggestion
* Shots can now be overridden for specific `qml.Snapshot` instances via a `shots` keyword argument.
```
How about that?

Reply:
```suggestion
* Shots can now be overridden for specific `qml.Snapshot` instances via a `shots` keyword argument.
  [(#7326)](https://github.com/PennyLaneAI/pennylane/pull/7326)
```

I think even if it's a circuit with 1 op and 1 snapshot instance that overrides device shots would be nice!